### PR TITLE
blog: Add "OCI Runtime Spec v1.1"

### PR DESCRIPTION
**Pull request by AkihiroSuda**
- https://github.com/opencontainers/runtime-spec/pull/1213

Let me merge the spec v1.1 PR and this blog PR simultaneously.
(ETA: 2023-07-20)

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
cc @opencontainers/runtime-spec-maintainers @caniszczyk @amye 
Whose LGTMs are needed for merging blog PRs?

File: content/posts/blog/2023-07-20-oci-runtime-spec-v1-1.md
Comment (AkihiroSuda) [inline content/posts/blog/2023-07-20-oci-runtime-spec-v1-1.md:59]: (Shamelessly citing my own project)

Comment (utam0k) [reply content/posts/blog/2023-07-20-oci-runtime-spec-v1-1.md:59]: 👍

**Review by tianon — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
@cyphar Thanks, applied your suggestions

**Review by utam0k — APPROVED** [review]

File: content/posts/blog/2023-07-20-oci-runtime-spec-v1-1.md
Comment (cyphar) [inline content/posts/blog/2023-07-20-oci-runtime-spec-v1-1.md:84]: This PR completely slipped my mind. I'll work on getting this merged and we can update the blog post after it's merged. Not a blocker for publishing of course.

**Review by cyphar — APPROVED** [review]
LGTM.

In the past I think we usually wanted to get approval from @caniszczyk or @amye before posting something to the OCI website. There isn't an official maintainer list (since I worked on the site in the past, I have merge rights but I suspect I'm not supposed to use them?).

**Comment by AkihiroSuda** [comment]
Changed the date to 2023-07-21

**Comment by AkihiroSuda** [comment]
- https://github.com/opencontainers/opencontainers.org/pull/130

Looks like spec maintainers can just merge blog PRs by themselves now?

**Pull request merged**
